index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Personal Portfolio Website</title>
+</head>
+
+<body>
+    <header>
+        <nav class="navigation">
+            <div class="logo">
+                <h3>Dhuruv</h3>
+            </div>
+            <div class="menu">
+
+            </div>
+        </nav>
+    </header>
+    <main>
+        <div class="headline-section">
+            <div class="illustration-mobile">
+                <img src="./icons/header-illsutration-mobile@2x.png" alt="">
+            </div>
+            <div class="heading-text">
+                <h1>Hey there. I’m Dhuruv</h1>
+                <h1 class="h-green">I’m a software developer</h1>
+            </div>
+            <div class="supporting-text">
+                <p> I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+            </div>
+            <ul class="social-icons">
+                <li>
+                    <a href="https://github.com/DhuruvSingh">
+                        <img src="./Social-icon-black/GitHub.png" alt="Github icon">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/dhuruv-singh-a1a51aa9/">
```

Comment:
- [ ] Please make sure to use the same margins  as in Figma design 
- [ ] Please I noticed that you are using Poppins font family instead of inter-as in Figma design 
- [ ] Please align the content of the headline section using flexbox 

| Your design | Figma | 
| ---------------- | -------- |
| ![image](https://user-images.githubusercontent.com/74744787/176781803-ea4bbcf1-c035-4dd2-baf4-195b3e4d8713.png) | ![image](https://user-images.githubusercontent.com/74744787/176781881-2d2f2795-05dd-499d-be8a-4721eab256cd.png) |

Reply:
@MeqdamAlqudah for margin, I checked all sizes and every size works perfectly except Galaxy Fold. for the specific size, I would have to use the media query if it is allowed in this project.